app/controllers/admin/base_controller.rb

```diff
@@ -5,7 +5,7 @@ class Admin::BaseController < ApplicationController
   layout 'admin'
   prepend_before_action :skip_slimmer
   prepend_before_action :authenticate_user!
-  before_action :require_signin_permission!
+  before_action :authenticate_user!
```

Comment:
Does this duplicate the behaviour on line 7? Or is there some Rails subtlety I don't know about?